Mage.Server.Plugins/Mage.Player.Human/src/mage/player/human/HumanPlayer.java

```diff
@@ -1916,13 +1916,17 @@ private boolean checkIfAttackersValid(Game game) {
         // check if enough attackers are declared
         // check if players have to be attacked
         Set<UUID> playersToAttackIfAble = new HashSet<>();
+        boolean mustAttackAPlayer = false;
```

Comment:
misleading name - effect says "must attack" so could be a planeswalker or battle

make sure to check your code for this possibility

Reply:
Good catch. My code isn't only checking for creatures attacking players (only that at least one creature is attacking at all), but I will modify the "can any creatures attack" check to check for any defenders at all, not just players

Reply:
I still don't like the variable name "mustAttackAPlayer"

Reply:
Ah i see, i misread.